pandas/io/formats/style.py

```diff
@@ -49,19 +48,21 @@ def _mpl(func):
 
 class Styler(object):
     """
-    Helps style a DataFrame or Series according to the
-    data with HTML and CSS.
+    A decorator class for a `DataFrame` object to provide an
```

Comment:
What's a "decorator class"? This like it'd be easily confused with python decorators, which I don't think you mean.

Reply:
By "decorator class" I mean the concept precisely defined on p. 175 of "Design Patterns" by Gamma et al (commonly referred to as Gang of Four, or GOF), which seems to precisely describe what Styler is: a class which contains additional state information to perform an auxiliary function on another class.  Is there a name we consistently use in our documentation to refer to such a concept?